build.cake

```diff
@@ -79,6 +69,43 @@ Task("Build")
         DotNetCoreBuild(solutionFullPath, settings);
     });
 
+Task("BuildArtifacts")
```

Comment:
This task copies the necessary files from the Visual Studio bin\Release directories to the artifacts\bin directories.